content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,350 @@
+=======
+Uruguay
+=======
+
+.. |DGI| replace:: :abbr:`DGI (Dirección General Impositiva)`
+.. |EDI| replace:: :abbr:`EDI (Electronic Data Interchange)`
+
+.. _uruguay/intro:
+
+Introduction
+============
+
+With the Uruguayan localization, you can generate electronic documents with its XML, fiscal folio,
+electronic signature and connection to tax authority Dirección General Impositiva (DGI) through
+Uruware.
+
+The supported documents are:
+
+- e-Invoice, e-Invoice Credit Note, e-Invoice Debit Note;
+- e-Ticket, e-Ticket Credit Note, e-Ticket Debit Note;
+- Export e-Invoice, Export e-Invoice Credit Note, Export e-Invoice Debit Note.
+
+The localization requires an Uruware account, which enables users to generate electronic documents
+within Odoo.
+
+Glossary
+--------
+
+The following terms are used throughout the Uruguayan localization:
+
+- **DGI**: *Dirección General Impositiva* is the government entity responsible for enforcing tax
+  payments in Uruguay.
+- **EDI**: *Electronic Data Interchange* refers to the sending of electronic documents.
+- **Uruware**: is the third-party organization that facilitates the interchange of electronic
+  documents between companies and the Uruguayan government.
+
```

Comment:
We can add:

- **CAE**: *Constancia de Autorización de Emisión* is a document requested from the tax authority's website to
   enable electronic invoice issuance.